.github/workflows/stale.yml

```
@@ -0,0 +1,40 @@
+name: Close stale issues and pull requests
+
+on:
+  schedule:
+    # Run once a day
+    - cron: "30 1 * * *"
+
+  push:
+    branches:
+      - stale
+
+jobs:
+  stale:
+    if: "github.repository == 'ESMValGroup/ESMValTool'"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          debug-only: true
+          days-before-issue-stale: 1080
+          days-before-pr-stale: 360
+          days-before-close: 60
+          ascending: true
+          operations-per-run: 1000
+          stale-issue-message: |
+            In order to maintain a backlog of relevant issues, we automatically label them as stale after 1080 days of inactivity.
+
+            If this issue is still important to you, please comment below to remove the stale label. Otherwise, this issue will be automatically closed in 60 days.
+          stale-pr-message: |
+            In order to maintain a backlog of relevant pull requests, we automatically label them as stale after 360 days of inactivity.
+
+            If this pull request is still important to you, please comment below to remove the stale label. Otherwise, this pull request will be automatically closed in 60 days. If this pull request only suffers from a lack of reviewers, please tag the @<span></span>ESMValGroup/technical-lead-development-team so they can help you find a suitable reviewer.
```

```suggestion
            If this pull request is still important to you, please comment below to remove the stale label. Otherwise, this pull request will be automatically closed in 60 days. If this pull request only suffers from a lack of reviewers, please add the tags "looking for scientific/technical reviewer" to the PR, and additionally tag the @<span></span>ESMValGroup/science-reviewers and @<span></span>ESMValGroup/tech-reviewers teams so they can help you find a suitable reviewer.
```
In the Tool I wouldn't want to tag the tech lead team for every PR here. I think that's what the reviewer teams are for (although we would have to do some housekeeping in those teams). Alternatively I would suggest the UserEngagementTeam over the TechLeadTeam in the tool. OBS-maintainers should be tagged automatically on cmorizers, but I find that we need to poke them separately usually to get them moving ;)

---

Changed inhttps://github.com/ESMValGroup/ESMValTool/pull/4100/commits/408c9f15af79cd93f346419bd24986c0cc704661

---

I'll not let that hold this PR up though, link that documentation part here and the TLT and SLT teams below and I'll click on accept!